### ISSUE TITLE
x86_64 appimage fails because of missing README.md

### ISSUE THREAD

**Issue by kseistrup** [fraction/oasis-desktop#3]
I installed the oasis-desktop-1.1.0-x86_64.AppImage on an ArchLinux box running Linux 5.6.3 x86_64. It never launched, presumably because of a missing README.md:

```
Error: ENOENT, node_modules/@fraction/oasis/README.md not found in /tmp/user/1000/.mount_oasis-MJFTbv/resources/app.asar
    at createError (electron/js2c/asar.js:111:17)
    at Object.fs.readFileSync (electron/js2c/asar.js:548:24)
    at Object.<anonymous> (/tmp/user/1000/.mount_oasis-MJFTbv/resources/app.asar/node_modules/@fraction/oasis/src/index.js:189:19)
    at Module._compile (internal/modules/cjs/loader.js:968:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:986:10)
    at Module.load (internal/modules/cjs/loader.js:816:32)
    at Module._load (internal/modules/cjs/loader.js:728:14)
    at Module._load (electron/js2c/asar.js:717:26)
    at Function.Module._load (electron/js2c/asar.js:717:26)
    at Module.require (internal/modules/cjs/loader.js:853:19) {
  code: 'ENOENT',
  errno: -2
```

----

It also complain about not finding the sodium bindings:

```
error loading sodium bindings: No native build was found for platform=linux arch=x64 runtime=electron abi=76 uv=1 libc=glibc
falling back to javascript version.
```

but that should hardly prevent it from running.

**Comment by christianbundy** [comment]
Thanks for this bug report! That isn't good at all. Maybe I need to add the readme as an asset for electron-builder? I've been anticipating that it might be nice to make the docs available in the UI, starting with the readme, but it looks like this is failing. I'm going to take a look now.

**Comment by christianbundy** [comment]
Resolved in #6 and published in the [latest release](https://github.com/fraction/oasis-desktop/releases/latest)! Tested both the AppImage and the DMG.
